Source/RimConnection/Managers/DropPodManager.cs

```diff
@@ -14,13 +14,12 @@ public static void createDropFromDef(ThingDef thingDef, int amount, string title
             if(newthing != null)
             {
                 var currentMap = Find.CurrentMap;
-                IntVec3 dropVector = DropCellFinder.TradeDropSpot(Find.CurrentMap);
-                TradeUtility.SpawnDropPod(dropVector, currentMap, newthing);
-
+                IntVec3? dropVector = DropCellFinder.TryFindSafeLandingSpotCloseToColony(Find.CurrentMap, IntVec2.Two);
+                TradeUtility.SpawnDropPod(dropVector ?? DropCellFinder.TradeDropSpot(Find.CurrentMap), currentMap, newthing);
                 if (showMessage)
                 {
                     string messageString = "RimConnectionPositiveDroppodMailBody".Translate(amount, title, desc);
-                    AlertManager.ResourceDropNotification(messageString, dropVector);
+                    AlertManager.ResourceDropNotification(messageString, dropVector ?? DropCellFinder.TradeDropSpot(Find.CurrentMap));
```

Comment:
This is going to instantiate a new dropcell compared to the one that you fallback to on line 18. I'd suggest getting the variable dropVector to be a definite value, i.e. do the `dropVector ?? DropCellFinder.TradeDropSpot(Find.CurrentMap)` before dropping or writing the messages

Reply:
The meaning of this code was in safety and returning to old behavior in case the TryFindSafeLandingSpotCloseToColony method will return null, in theory this is possible, and I decided to play it safe. But in principle, this is not necessary.

Reply:
The function signature didn't seem to say it could return null and looks like it already falls back to "Can't find spot, dropping stuff randomly on the map". All good 😄, thank you very much for the PR